packages/page-accounts/test/pages/accountsPage.tsx

```diff
@@ -25,12 +25,48 @@ let queueExtrinsic: (value: PartialQueueTxExtrinsic) => void;
 class NotYetRendered extends Error {
 }
 
-export type AccountRow = HTMLElement;
+// utility wrapper over an account item in accounts table, serves basic assertions about an account row
+export class AccountRow {
+  public primaryRow: HTMLElement;
+  public detailsRow: HTMLElement;
+
+  constructor (primaryRow: HTMLElement, detailsRow: HTMLElement) {
+    this.primaryRow = primaryRow;
+    this.detailsRow = detailsRow;
+  }
+
+  async assertBalancesTotal (expected: Balance): Promise<void> {
+    const balanceActual = await within(this.primaryRow).findByTestId('balance-summary');
+    const balanceExpected = formatBalance(expected, { decimals: 12, withUnit: true });
+
+    expect(balanceActual).toHaveTextContent(balanceExpected);
+  }
+
+  async assertBalancesDetails (expected: {name: string, amount: Balance}[]): Promise<void> {
+    for (const expectedBalanceDetailsItem of expected) {
+      const labelElement = await within(this.detailsRow).findByText(expectedBalanceDetailsItem.name);
+      const balanceElement = labelElement.nextSibling;
+      const amount = formatBalance(expectedBalanceDetailsItem.amount, { decimals: 12, withUnit: true });
```

Comment:
I think we need `forceUnit: '-'` here as well because we it do in `FormatBalance`[1], which is what displays in tests. If we skip it, then the text content will differ for some amounts; I encountered this in my own tests, but don't remember how to reproduce (which values give different results when this param is not specified).
Since I use this code as well, maybe we should also refactor it into a separate function, like `defaultFormatBalance`?

[1] https://github.com/polkadot-js/apps/blob/master/packages/react-query/src/FormatBalance.tsx#L60

Reply:
That was great advice! Thx! For now I refactored outside the class, to a common fn in AccountPage and AccountRow.